bindgen/features.rs

```diff
@@ -86,35 +87,35 @@ macro_rules! rust_target_base {
     ( $x_macro:ident ) => {
         $x_macro!(
             /// Rust stable 1.0
-            => Stable_1_0 => 1.0;
+            #[deprecated] => Stable_1_0 => 1.0;
             /// Rust stable 1.17
             ///  * Static lifetime elision ([RFC 1623](https://github.com/rust-lang/rfcs/blob/master/text/1623-static.md))
-            => Stable_1_17 => 1.17;
+            #[deprecated] => Stable_1_17 => 1.17;
             /// Rust stable 1.19
             ///  * Untagged unions ([RFC 1444](https://github.com/rust-lang/rfcs/blob/master/text/1444-union.md))
-            => Stable_1_19 => 1.19;
+            #[deprecated] => Stable_1_19 => 1.19;
             /// Rust stable 1.20
             ///  * Associated constants ([PR](https://github.com/rust-lang/rust/pull/42809))
-            => Stable_1_20 => 1.20;
+            #[deprecated] => Stable_1_20 => 1.20;
             /// Rust stable 1.21
             ///  * Builtin impls for `Clone` ([PR](https://github.com/rust-lang/rust/pull/43690))
-            => Stable_1_21 => 1.21;
+            #[deprecated] => Stable_1_21 => 1.21;
             /// Rust stable 1.25
             ///  * `repr(align)` ([PR](https://github.com/rust-lang/rust/pull/47006))
-            => Stable_1_25 => 1.25;
+            #[deprecated] => Stable_1_25 => 1.25;
             /// Rust stable 1.26
             ///  * [i128 / u128 support](https://doc.rust-lang.org/std/primitive.i128.html)
-            => Stable_1_26 => 1.26;
+            #[deprecated] => Stable_1_26 => 1.26;
             /// Rust stable 1.27
             ///  * `must_use` attribute on functions ([PR](https://github.com/rust-lang/rust/pull/48925))
-            => Stable_1_27 => 1.27;
+            #[deprecated] => Stable_1_27 => 1.27;
```

Comment:
Can we use `#[deprecated = "<message>"]` here? Would be nice to get a proper rustc warning at compile time rather than a `warn!()` at runtime.

Reply:
Sounds good to me.